instrumentation/opentelemetry-instrumentation-aiohttp-server/src/opentelemetry/instrumentation/aiohttp_server/__init__.py

```diff
@@ -296,7 +294,22 @@ class AioHttpServerInstrumentor(BaseInstrumentor):
     """
 
     def _instrument(self, **kwargs):
+        tracer_provider = kwargs.get("tracer_provider", None)
```

Comment:
Thanks for working on this!

Please could you also [update the readthedocs](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/34db73ebfd97761ba65523c3f07315ce2974dc78/instrumentation/opentelemetry-instrumentation-aiohttp-server/src/opentelemetry/instrumentation/aiohttp_server/__init__.py#L19-L38) to describe the new optional tracer_provider.